core/src/main/java/com/valtech/aem/saas/core/fulltextsearch/SearchModelImpl.java

```diff
@@ -220,6 +225,21 @@ public String getId() {
         return Optional.empty();
     }
 
+    private Optional<String> createTrackingUrl() {
+        if (Optional.ofNullable(searchCAConfigurationModel)
+                    .filter(SearchCAConfigurationModel::isSearchResultItemTrackingEnabled)
+                    .isPresent()) {
+            return Optional.ofNullable(request)
+                           .map(r -> pathTransformer.map(r, resource.getPath()))
+                           .map(url -> String.format("%s.%s.%s",
+                                                     url,
+                                                     SearchResultItemTrackingServlet.SEARCH_RESULT_ITEM_TRACKING_SELECTOR,
+                                                     SearchResultItemTrackingServlet.SEARCH_RESULT_ITEM_TRACKING_EXTENSION));
+        }
+        log.info("Search Result Item Tracking is not enabled. To enable it, please check context aware SearchConfiguration.");
```

Comment:
we shouldn't log this on info level as it will flood our logfiles. please change to debug